controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -489,6 +468,38 @@ func (r *StorageConsumerReconciler) reconcileConsumerConfigMap(
 				return err
 			}
 		}
+
+		if controllerRef != nil && controllerRef.UID == r.storageConsumer.UID {
+			if consumerConfigMap.Data == nil {
+				consumerConfigMap.Data = map[string]string{}
+			}
+			defaultConsumerResourceNames := util.GetStorageConsumerDefaultResourceNames(
+				r.storageConsumer.Name,
+				string(r.storageConsumer.UID),
+				availableServices,
+			)
+			for key := range defaultConsumerResourceNames {
+				consumerConfigMap.Data[key] = cmp.Or(
+					strings.Trim(consumerConfigMap.Data[key], " "),
+					defaultConsumerResourceNames[key],
+				)
+			}
```

Comment:
This code should also run in case there is no controller ref. You should move it into the first if case and argument the condition to 
`if controllerRef == nil || controllerRef == r.storageConsumer.UID`